core/src/test/kotlin/com/tschuchort/compiletesting/KotlinJsCompilationTests.kt

```diff
@@ -78,27 +78,6 @@ class KotlinJsCompilationTests {
 		assertThat(jsFile.readText()).contains("function KSource_0() {")
 	}
 
-	@Test
```

Comment:
Unfortunately I’m not sure why this no longer works, it fails with `kotlinx.browser` not being resolved. Maybe this is no longer supported? But [the KDoc](https://kotlinlang.org/api/latest/jvm/stdlib/kotlinx.browser/) seems to indicate it should still work 🤔 The source is still [there](https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/js/src/kotlinx/browser/declarations.kt). Any idea?